coalib/settings/ConfigurationGathering.py

```diff
@@ -106,6 +106,12 @@ def warn_nonexistent_targets(targets, sections, log_printer):
                 "Thus it cannot be executed.".format(section=target))
 
 
+def warn_config_absent(sections, argument, log_printer):
```

Comment:
There's no docstring; everything else has one so I guess we'd like one here too?